common/include/common/binary/BinaryBuffer.hpp

```diff
@@ -121,21 +122,33 @@ template <>
 BinaryBuffer::BinaryBuffer(float f);
 template <>
 BinaryBuffer::BinaryBuffer(char c);
+/*template <>
+BinaryBuffer::BinaryBuffer(uint64_t c);*/
```

Comment:
I suppose this is commented out due to the fact, this conflicts with `size_t` one. Remove commented out code or make it work. In this case I suppose, you can only remove it.